UiPath.FreeRdpClient/UiPath.SessionTools.Tests/ProcessRunnerTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Diagnostics;
+using System.Reflection;
+
+namespace UiPath.SessionTools.Tests;
+
+[Trait("Subject", nameof(ProcessRunner))]
+public class ProcessRunnerTests
```

Comment:
teaser:) : you could also add/move as integration tests that checks the actual output logs
maybe it is simpler than having the tests with fakes